plugins/org.python.pydev.refactoring/tests/python/coderefactoring/extractmethod/testExtractMethod0_grammar38.py

```diff
@@ -0,0 +1,23 @@
+class A:
```

Comment:
This test isn't really related to the issue...

The error is when refactoring the construct below:

```
        try:
            a = 10
        except:
            e: ValueError
```

So, that construct should be tested.